src/main/scala/nl.knaw.dans.easy.multideposit/SplitMultiDepositApp.scala

```diff
@@ -26,11 +27,12 @@ import nl.knaw.dans.easy.multideposit.parser.MultiDepositParser
 import nl.knaw.dans.lib.error.{ CompositeException, TraversableTryExtensions }
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 
-import scala.util.control.NonFatal
+import scala.language.postfixOps
```

Comment:
where do you need this one?

Reply:
IntelliJ complained about my dot-less calls to `isRegularFile` and `isExecutable` on line 126-7.